server/controllers/patients/getAllPatientData.js

```diff
@@ -0,0 +1,37 @@
+const { boomify } = require('../../utils');
+
+const {
+  getHistoryLogs,
+  getPatientProfileData,
+} = require('../../database/queries');
+
+const getAllProfileData = async (req, res, next) => {
+  try {
+    const { patientId } = req.params;
+    const {
+      rows: [profile],
+    } = await getPatientProfileData({ patientId });
+    if (!profile) {
+      return next(boomify(400, 'Invalid id', 'Please send a correct one'));
+    }
+
+    const { rows: history } = await getHistoryLogs({ patientId });
+    const balance = history.reduce(
+      (sum, obj) => sum + parseFloat(obj.balance),
+      0,
+    );
+    return res.status(201).json({
```

Comment:
Why did you send statusCode 201?
```suggestion
    return res.json({
```

Reply:
sorry I thought it meant sometuhing else